src/RoomInfoListItem.cc

```diff
@@ -77,12 +77,12 @@ RoomInfoListItem::RoomInfoListItem(RoomInfo info, QWidget *parent)
 	avatarLayout_->addWidget(roomAvatar_);
 
 	roomName_ = new QLabel(info_.name(), textWidget_);
-	roomName_->setMaximumSize(parent->width() - 10, 20);
+	roomName_->setMaximumSize(parent->width() - max_height_, 20);
```

Comment:
I don't think those are necessary, it just squashes the labels further inside. The 
```
setMaximumSize(parent->width(), max_height_);
``` 
line, was probably the bugfix. Could you confirm on this?